llvm/lib/Target/X86/X86Subtarget.cpp

```diff
@@ -91,9 +91,8 @@ X86Subtarget::classifyLocalReference(const GlobalValue *GV) const {
       if (CM == CodeModel::Large)
```

Comment:
Is this condition correct? It seems like it won't handle globals explicitly marked with `code_model "small"` correctly.